keyboards/lime/rev1/config.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0666
```

Comment:
I am genuinely surprised that this isn't taken already.  Though, the low number worries me.  It doesn't need to be changed, though.

Reply:
To be honest, me too. I'll change this in the future.